fpgaperf.py

```diff
@@ -404,14 +402,24 @@ def run(
     t.strategy = strategy
     t.seed = seed
     t.carry = carry
+
+    # Constraint files shall be in their directories
```

Comment:
This wrapping seems weird?

Reply:
@mithro You mean the constraints file locations?